README.md

```diff
@@ -1,10 +1,24 @@
 <p align="center">
-  <img src="docs/assets/dot-matrix-logo.png" alt="Nitric Logo"/>
+  <img src="docs/assets/nitric-logo.svg" width="120" alt="Nitric Logo"/>
```

Comment:
Can you make this a link to the nitric site too?

Reply:
done